source/features/clone-branch.tsx

```diff
@@ -0,0 +1,89 @@
+import delegate, {DelegateEvent} from 'delegate-it';
+import gitBranch from 'octicon/git-branch.svg';
+import insertTextTextarea from 'insert-text-textarea';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+async function cloneBranch(event: DelegateEvent<MouseEvent, HTMLAnchorElement>): Promise<void> {
+	const currentTarget = event.delegateTarget;
+	const branchElement = currentTarget.closest<HTMLAnchorElement>('[data-branch-name]');
+	const spinner = select('.js-loading-spinner', branchElement!)!;
+	spinner.hidden = false;
+	currentTarget.hidden = true;
+
+	let newBranchName = prompt('Enter the new branch name')?.trim();
+	if (!newBranchName) {
+		return;
+	}
+
+	const getBranchInfo = await api.v3(`repos/${getRepoURL()}/git/refs/heads/${branchElement!.dataset.branchName!}`);
+	let createBranch = await createNewBranch(newBranchName, getBranchInfo);
```

Comment:
`createBranch` is not a function, so it should not be named `create*`. 

Perhaps:

```js
result = await createNewBranch(newBranchName, (await currentBranch).object.sha); 
```

Reply:
What about getting the branch info until the popup shows? They need to wait for that. (Like you asked for [here](https://github.com/sindresorhus/refined-github/pull/2896#discussion_r395136759))

Reply:
How’s that related to this review comment?

Reply:
What you have now is good 👍

Reply:
Sorry I was doing 2 things at once